Cannot build frida because valac segfaults

Buiilding from scratch is not possible for me because valac cannot execute:

    $ make -f Makefile.sdk.mk
    ...
    meson.build:1:0: ERROR:  Unknown compiler(s): [['/home/grayfox/reversing/frida/build/fs-linux-x86_64-valac']]

    A full log can be found at /home/grayfox/reversing/frida/build/fs-tmp-linux-x86_64/libgee/meson-logs/meson-log.txt
    make: *** [Makefile.sdk.mk:287: build/fs-tmp-linux-x86_64/libgee/build.ninja] Error 1
    [grayfox@shadow-moses ~/reversing/frida frida-linux-x86_64]
    $ /home/grayfox/reversing/frida/build/fs-linux-x86_64-valac
    Segmentation fault (core dumped)

Seems to be a duplicate of #706  (which is closed without comment)

OS: Ubuntu 18.10

--- a/releng/setup-env.sh
+++ b/releng/setup-env.sh
@@ -795,7 +795,7 @@ fi
   echo "export ACLOCAL=\"$ACLOCAL\""
   echo "export CONFIG_SITE=\"$CONFIG_SITE\""
   echo "unset LANG LC_ALL LC_COLLATE LC_CTYPE LC_MESSAGES LC_NUMERIC LC_TIME"
-  echo "export LC_ALL=en_US.UTF-8"
+  echo "export LC_ALL=en_US"
 ) > $env_rc
 
 case $host_platform in

